src/SprykerEco/Zed/UnzerApi/Business/Api/Request/UnzerApiRequestConstants.php

```diff
@@ -423,4 +423,9 @@ interface UnzerApiRequestConstants
      * @var string
      */
     public const PARAM_ITEMS = 'items';
+
+    /**
+     * @var string
+     */
+    public const PARAM_AVAILABLE_PAYMENT_METHODS = 'availablePaymentTypes';
```

Comment:
Looks like a part of [module config](https://spryker.atlassian.net/wiki/spaces/CORE/pages/478052552/Application+Namespace+Conventions#ApplicationNamespaceConventions-ModuleConfig) not [constants](https://spryker.atlassian.net/wiki/spaces/CORE/pages/478052552/Application+Namespace+Conventions#ApplicationNamespaceConventions-Constants).

Reply:
Will not be changed as discussed